docs/source/bft_configuration.md

```diff
@@ -0,0 +1,221 @@
+Apart from the (already discussed) consenters, the SmartBFT channel configuration has a section which relates to protocol specific knobs.
+It is possible to change these values dynamically at runtime, described in the Reconfiguration section below.
+
+* `RequestBatchMaxCount`: The maximal number of requests in a batch. A request batch that reaches this count is proposed immediately.
+* `RequestBatchMaxBytes`: The maximal total size of requests in a batch, in bytes. This is also the maximal size of a single request. A request batch that reaches this size is proposed immediately.
+* `RequestBatchMaxInterval`: The maximal time interval a request batch can wait before it is proposed. A request batch is accumulating requests until `RequestBatchMaxInterval` had elapsed from the time the batch was first created (i.e. the time the first request was added to it), or until it is of count `RequestBatchMaxCount`, or it reaches `RequestBatchMaxBytes`, whichever occurs first.
+* `IncomingMessageBufferSize`: The size of the buffer holding incoming messages before they are processed (maximal number of messages).
+* `RequestPoolSize` : The number of pending requests retained by the node. The `RequestPoolSize` is recommended to be at least double (x2) the `RequestBatchMaxCount`. This cannot be changed with a reconfiguration.
+* `RequestForwardTimeout`: Is started from the moment a request is submitted, and defines the interval after which a request is forwarded to the leader.
+* `RequestComplainTimeout`: Is started when `RequestForwardTimeout` expires, and defines the interval after which the node complains about the view leader.
+* `RequestAutoRemoveTimeout`: Is started when `RequestComplainTimeout` expires, and defines the interval after which a request is removed (dropped) from the request pool.
+* `ViewChangeResendInterval`: Defines the interval after which the ViewChange message is resent.
+* `ViewChangeTimeout`: Is started when a node first receives a quorum of `ViewChange` messages, and defines the interval after which the node will try to initiate a view change with a higher view number.
+* `LeaderHeartbeatTimeout`: Is the interval after which, if nodes do not receive a "sign of life" from the leader, they complain about the current leader and try to initiate a view change. A sign of life is either a heartbeat or a message from the leader.
+* `LeaderHeartbeatCount`: Is the number of heartbeats per `LeaderHeartbeatTimeout` that the leader should emit. The heartbeat-interval is equal to: `LeaderHeartbeatTimeout/LeaderHeartbeatCount`.
+* `CollectTimeout`: Is the interval after which the node stops listening to `StateTransferResponse` messages, stops collecting information about view metadata from remote nodes.
+
+## Reconfiguration
+
+The Smart BFT orderer supports dynamic (meaning, while the channel is being serviced) addition and removal of nodes, and configuration changes.
+The only configuration parameter which cannot be changed is the `RequestPoolSize`.
```

Comment:
cannot be changed *dynamically*. We should be able to change it with a restart, no?